src/component/ProductCard.js

```diff
@@ -25,7 +37,7 @@ function ProductCard( {prod}) {
                 </div>
               </div>
               <div className="footer-card">
-                <div className="icons txt-lg bold-font">
+                <div className="icons txt-lg bold-font" onClick={() => addWishlistHandler(prod)}>
```

Comment:
I think you should have the icon behave like a toggle, meaning that add to wishlist will show a filled heart icon & clicking on that again removes it from the wishlist & shows an empty heart icon. So in your addWishlistHandler also, when the item is already present in the wishlist, clicking the filled heart removes it from the wishlist

Reply:
Thank you, Aishwarya for the suggestion done applied the Toggle functionality on the button